src/stores/HomeStore.js

```diff
@@ -124,6 +125,11 @@ class HomeStore {
       } catch(e) {
         this.tokenName = this.homeWeb3.utils.hexToAscii(await getName(alternativeContract)).replace(/\u0000*$/, '')
       }
+      try {
+        this.tokenDecimals = await getDecimals(this.tokenContract)
+      } catch(e) {
+        this.tokenDecimals = this.homeWeb3.utils.hexToAscii(await getDecimals(alternativeContract)).replace(/\u0000*$/, '')
```

Comment:
Similar to `ForeignStore.js` there is no need to call `getDecimals` with `alternativeContract`. It is only needed for `name` and `symbol`

Reply:
ok I remove the alternativeContract call in 6669067